openmls/src/group/mls_group/extension.rs

```diff
@@ -0,0 +1,52 @@
+//! MLS group context extensions
+//!
+//! Contains all the method related to modifying a group's extensions
+
+use openmls_traits::signatures::Signer;
+
+use crate::prelude::{hash_ref::ProposalRef, CreateGroupContextExtProposalError};
+
+use super::*;
+
+impl MlsGroup {
```

Comment:
You also need to add a method to create a commit with a GCE proposal. Let's call it `update_extensions` (unless you have a better idea)